### ISSUE TITLE
RHEL9: unable to install using setup-repos.sh

### ISSUE THREAD

**Issue by defung** [webmin/webmin#1959]
I ran the provided "setup-repos.sh", which installed fine. However, when I ran "dnf install webmin", it failed with the following error:

```
GPG key at file:///etc/pki/rpm-gpg/RPM-GPG-KEY-webmin-developers (0x8916F2A2) is already installed
The GPG keys listed for the "Webmin - noarch" repository are already installed but they are not correct for this package.
Check that the correct key URLs are configured for this repository.. Failing package is: usermin-1.861-1.noarch
```

**Comment by jcameron** [comment]
Did you previously have webmin installed on that system, or is it a fresh machine?

**Comment by defung** [comment]
This is not a new machine, but I have never installed webmin before.

I tested it in a `rockylinux:9` docker container, and installation of webmin succeeds, but it didn't have a dependency on usermin. I'm not sure why my RHEL9 instance is pulling in usermin as a dependency.

However, using the `rockylinux:9` container, I did get the same error when explicitly installing usermin.

Perhaps this is an issue with usermin, but I think it wasn't signed correctly.

I'll close this and create an issue in the usermin repo.
